pkg/cmd/template/schema_test.go

```diff
@@ -393,7 +418,31 @@ not_in_schema: "this should fail the type check!"
 			files.MustNewFileFromSource(files.NewBytesSource("schema.yml", []byte(schemaYAML))),
 			files.MustNewFileFromSource(files.NewBytesSource("values.yml", []byte(dataValuesYAML))),
 		})
-		expectedErr := "Typechecking violations found: [Expected node at values.yml:2 to be nil, but was a *yamlmeta.Map]"
+		expectedErr := "data values were found in data values file(s), but schema (schema.yml) has no values defined\n"
+		expectedErr += "(hint: define matching keys from data values files(s) in the schema, or do not enable the schema feature)"
+
+		assertYTTWorkflowFailsWithErrorMessage(t, filesToProcess, expectedErr)
+	})
+	t.Run("multiple data values given but schema is empty", func(t *testing.T) {
```

Comment:
What is the value added by this test? found it difficult to see the difference between this and the one above, "data values is given but schema is empty"

Reply:
It mainly was added to TDD. Originally, this error contained the values file name but only one if multiple were supplied. It helps reinforce the design of the error message.